### PR TITLE
Push to docker hub for use by idp-id-broker

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

**Review by jason-jackson — APPROVED** [review]

File: codeship-steps.yml [hunk 1]
```diff
@@ -11,3 +11,21 @@
   service: tracker
   tag: ^(main|develop)
   command: "true"
+
+- name: push_develop
+  service: app
+  type: push
+  image_name: silintl/serverless-mfa-api-go
+  image_tag: "latest"
+  tag: develop
```
Comment (forevermatt) [inline codeship-steps.yml:20]: For pushing develop and tagged versions, I think we normally tag the image by the branch name (which includes version tags):
```suggestion
  image_tag: "{{.Branch}}"
  exclude: main
```
... then reserve the tag "latest" for the `main` (or `master`) branch.

(An example of this is [idp-id-broker's codeship-steps.yml](https://github.com/silinternational/idp-id-broker/blob/develop/codeship-steps.yml) file.)

File: codeship-steps.yml [hunk 2]
```diff
@@ -11,3 +11,21 @@
...
+  registry: https://index.docker.io/v1/
+  encrypted_dockercfg_path: dockercfg.encrypted
+
+- name: push_production
+  service: app
+  type: push
+  image_name: silintl/serverless-mfa-api-go
+  image_tag: "prod"
```
Comment (forevermatt) [inline codeship-steps.yml:28]:
```suggestion
  image_tag: "latest"
```

**Review by forevermatt — CHANGES_REQUESTED** [review]
Two small changes request, otherwise looks great. 👍

File: codeship-steps.yml (hunk 1 above)
Comment (Baggerone) [reply codeship-steps.yml:20]: OK. I just didn't want it to push an image that hadn't been PR reviewed.

File: codeship-steps.yml (hunk 2 above)
Comment (Baggerone) [reply codeship-steps.yml:28]: I don't mind doing it that way.  I just didn't like the idea of the default tag "latest" subtly referring to "prod". You have to re-read this file any time you want to check what "latest" means in this repo.

Comment (forevermatt) [reply codeship-steps.yml]: Gotcha. The general practice for Docker images seems to be that "latest" always refers to prod, from what I understand.

File: codeship-steps.yml (hunk 1 above)
Comment (forevermatt) [reply codeship-steps.yml]: Ahh, that makes sense. If you want to limit it down like that, I think you can do something like `tag: (develop|main)`. I'm fine with either what's already here or with that approach. I don't typically see any value in pushing images for feature branches (typically), so feel free to make that change if you're so inclined. I'll approve either way.

**Review by forevermatt — APPROVED** [review]

**Pull request merged**